LICENSE.mustache

```diff
@@ -1,44 +1,105 @@
-   this software.
+## Copyright
 
-4. Ensure everyone who gets a copy of this software from you, in
-   source code or any other form, gets the text of this license
-   and the contributor and source code lines above.
+The contributor licenses you to do everything with this
+software that would otherwise infringe their copyright
+in it.
 
-5. Do not make any legal claim against anyone accusing this
-   software, with or without changes, alone or with other
-   software, of infringing any patent claim.
+## Notices
 
-To contribute software, publish all its source code, in the
-preferred form for making changes, through a freely accessible
-distribution system widely used for similar source code, and
-license contributions not already licensed to the public on terms
-as permissive as this license accordingly.
+You must ensure that everyone who gets a copy of any
+part of this software from you, with or without changes,
+also gets the text of this license and the contributor
+and source code lines above.
 
-You are excused for unknowingly breaking 1, 2, or 3 if you
-contribute as required, or stop doing anything requiring this
-license, within 30 days of learning you broke the rule.
+## Patent
 
-**As far as the law allows, this software comes as is, without
-any warranty, and the contributor will not be liable to anyone
-for any damages related to this software or this license, for any
-kind of legal claim.**
+The contributor licenses you to do everything with this
+software that would otherwise infringe any patent claims
+they can license or become able to license.
+
+## Defense
+
+Do not make any legal claim against anyone accusing this
+software, with or without changes, alone or with other
+software, of infringing any patent claim.
+
+## Reliability
+
+The contributor cannot revoke this license.
+
+## Copyleft
+
+With the exception of prototypes:
+
+1.  Contribute all changes and additions you make to
+    this software
+
+2.  If you combine this software with other software,
+    contribute that software.
+
+3.  Contribute software you develop, deploy, monitor,
+    or run with this software.
+
+4.  When in doubt, contribute the software.
+
+## Prototypes
+
+You need not contribute prototype changes, extensions,
+or other software that you do not end up using for more
+than fourteen calendar days, sharing with anyone else,
+or using to provide a service to anyone else.
+
+## Contributing
+
+When this license requires you to contribute software:
+
+1.  Publish all source code for that software, in the
+    preferred form for making changes, through a freely
+    accessible distribution system widely used for similar
+    source code, so the contributors and others can find
+    and copy it.
+
+2.  Ensure that each part of that source code is available
+    to the public under a license at least as permissive
+    as this one.
+
+3.  Take these steps within thirty calendar days of
+    creating or using the software for the first time.
```

Comment:
It definitely overlaps, especially since the deadlines---14 days for prototypes and 30 days generally---aren't the same. I will push a commit to lengthen the prototypes deadline to 30 days.

Reply:
I should probably just avoid "creating or using" altogether. It's overly specific. What we really mean is "doing anything that requires a license". Compare the Excuse section:

> You are excused for unknowingly breaking [Copyleft](#copyleft) if you contribute as required, or stop doing anything requiring this license, within thirty days of learning you broke the rule.

@zkat, you just helped me spot a drafting improvement that I'd looked past for years. It's really small, and poses very little risk of creating needless conflict. It's pretty technical. But it's there, and I'm about to disappear it!